source/crud/tests/README.rst

```diff
@@ -137,6 +149,13 @@ single operation. Notable differences from the current format are as follows:
   defined under the ``tests[i].outcome.error`` and ``tests[i].outcome.result``
   fields.
 
+- Instead of a top-level ``runOn`` field, server requirements are denoted by
+  separate top-level ``minServerVersion`` and ``maxServerVersion`` fields. The
+  minimum server version is an inclusive lower bound for running the test. The
+  maximum server version is an exclusive upper bound for running the test. If a
+  field is not present, it should be assumed that there is no corresponding bound
+  on the required server version.
```

Comment:
Is it possible for a test to use runOn and minSV/maxSV simultaneously? I think this is a difficult requirement to fulfill. If possible any given file should use one or the other.

Reply:
No test should use both simultaneously. This paragraph should make it clear that they are mutually exclusive:

> Instead of a top-level ``runOn`` field, server requirements are denoted by separate top-level ``minServerVersion`` and ``maxServerVersion`` fields.

This section is describing "Legacy Test Format for Single Operations" for the v1 directory and noting differences from the earlier "Test Format" section, which applies to the v2 directory.

Reply:
I think I misread that as "server requirements *can be* denoted ...", thanks.